treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -43,23 +43,36 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity):
     extra_properties = {}
     for k in signature_props.keys():
         if k not in ['option_collection_hash', 'machine_platform',
-                     'test', 'suite']:
+                     'test', 'suite', 'has_subtests', 'parent_signature']:
             extra_properties[k] = signature_props[k]
 
     repository = Repository.objects.get(name=project_name)
 
+    defaults = {
+        'test': signature_props.get('test', ''),
+        'suite': signature_props['suite'],
+        'option_collection': option_collection,
+        'platform': platform,
+        'framework': framework,
+        'extra_properties': extra_properties,
+        'has_subtests': signature_props.get('has_subtests', False),
+        'last_updated': datetime.datetime.fromtimestamp(0)
+    }
+
+    if parent_hash:
+        # the parent PerformanceSignature object should have already been created
+        try:
+            defaults['parent_signature'] = PerformanceSignature.objects.get(
+                signature_hash=parent_hash)
```

Comment:
You should also specify repository and framework arguments to this, as it is possible for the same signature to occur in different repositories and/or frameworks.